### PR TITLE
Correct Schema for StringFormat

### PR DESCRIPTION
**Pull request by Surendhar1997**
Corrected schema validator for String format with type string missing in schema and added testcases for both in kirun java and kirun js.

### CODE REVIEW

File: kirun-java/pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	</parent>
 	<groupId>com.fincity.nocode</groupId>
 	<artifactId>kirun-java</artifactId>
-	<version>1.4.12</version>
+	<version>1.4.13</version>
```
Comment (serialcoder13) [inline kirun-java/pom.xml:12]: I recently changed the version please upgrade it to 1.4.14

**Review by serialcoder13 — CHANGES_REQUESTED** [review]
Please change the version rest is good.

File: kirun-java/pom.xml (hunk 1 above)
Comment (serialcoder13) [reply kirun-java/pom.xml:12]: In bot pom and package

Comment (serialcoder13) [reply kirun-java/pom.xml:12]: *both

Comment (Surendhar1997) [reply kirun-java/pom.xml]: updated

**Review by serialcoder13 — APPROVED** [review]
LGTM

**Pull request merged**